sdk/helper/base62/base62.go

```diff
@@ -48,3 +53,65 @@ func RandomWithReader(length int, reader io.Reader) (string, error) {
 		}
 	}
 }
+
+// Encode encodes bytes to base62.  This does *not* scale linearly with input as base64, so use caution
```

Comment:
This seems like a caution that will be  a) unknown to the user,  b) leave them stuck if they do know about it. Is there an alternative but safe/stable version of these processes such that maybe we conditionally use one or the other alg based on input size?

Reply:
Apparently not.  There seems to be no solution to 62^a = 256^b unlike with say base64, so you can't iteratively process the input to keep the number sizes small.  Base62 is in short pretty weird.  I've found other implementations on the web that do strange things to make it work, but they don't decode in a way that's compatible with what we produce in Random*

Reply:
We could certainly rename the functions or even have an arbitrary limit (or both).  Say EncodeSmall, with a 1000 character limit or something.